buildSrc/src/main/java/com/google/firebase/gradle/plugins/publish/PublishingPlugin.java

```diff
@@ -194,6 +218,19 @@ public void apply(Project project) {
             });
   }
 
+  private List<String> readReleaseConfigFile(String publishConfigurationFilePath) {
+    try (Stream<String> stream = Files.lines(Path.of(publishConfigurationFilePath))) {
+      return stream
+          .dropWhile((line) -> !line.equals("[modules]"))
```

Comment:
This is a smart way to do it, just drop all elements until you hit modules, skip over the signifier, and return the remaining content as a list.

Good job!